include/f_core/net/transport/c_udp_socket.h

```diff
@@ -0,0 +1,55 @@
+#ifndef C_UDP_SOCKET_H
+#define C_UDP_SOCKET_H
+
+#include <cstdint>
+#include <f_core/net/c_transciever.h>
+
+class CIPv4;
+
+class CUdpSocket : public CTransceiver {
+public:
+    /**
+     * Constructor
+     * @param ip IP address instance to bind to
+     * @param srcPort Source port to bind to
+     * @param dstPort Destination port to send to
+     */
+    CUdpSocket(CIPv4 &ipv4, uint16_t srcPort, uint16_t dstPort);
+
+    /**
+     * Destructor
+     */
+    ~CUdpSocket();
+
+    /**
+     * See parent docs
+     */
+    int TransmitSynchronous(const void *data, size_t len) override;
```

Comment:
You say to see parent docs and yet there are not parent docs.... Curious

Reply:
Fixed